### PR TITLE
Working on making the form UX work rather better

### PR DESCRIPTION
**Pull request by rossbruniges**
Relating to a couple of bugs (#36 and #38) raised by @dethe

### CODE REVIEW

File: gameon/users/templates/users/profile_edit.html [hunk 1]
```diff
@@ -62,4 +63,10 @@ <h1 class="busta shout">
         </form>
     </div>
 </div>
+{% endblock %}
+
+{% block page_js %}
+<script src="{{ static('base/js/libs//jquery.textareaCounter.plugin.js') }}"></script>
```
Comment (andrewhayward) [inline gameon/users/templates/users/profile_edit.html]: Random double slash?

File: gameon/submissions/forms.py [hunk 2]
```diff
@@ -5,10 +5,18 @@
...
     ('False', "No, I don't want to submit my game to the marketplace"))
 
 entry_widgets = {
+    'url': forms.TextInput(attrs={'aria-describedby': 'info_url'}),
+    'description': forms.Textarea(attrs={'aria-describedby': 'info_description',
+        'data-maxlength': '1000'}),
+    'video_url': forms.TextInput(attrs={'aria-describedby': 'info_video_url'}),
+    'team_members': forms.Textarea(attrs={'aria-describedby': 'info_team_members',
+        'data-maxlength': '250'}),
+    'team_desciption': forms.Textarea(attrs={'aria-describedby': 'info_team_desciption',
```
Comment (andrewhayward) [inline gameon/submissions/forms.py:18]: "Team _description_"?

File: gameon/submissions/models.py [hunk 3]
```diff
@@ -75,9 +75,9 @@ class Entry(models.Model):
     team_name = models.CharField(max_length=255, verbose_name=_(u'Team name'),
         blank=True)
     team_members = models.TextField(verbose_name=_(u'Members'),
-        validators=[MaxLengthValidator(300)], blank=True)
+        validators=[MaxLengthValidator(250)], blank=True)
     team_desciption = models.TextField(verbose_name=_(u'Description'),
```
Comment (andrewhayward) [inline gameon/submissions/models.py:79]: Probably a pain to change this now - but I assume that should be `team_description`?

File: gameon/submissions/templates/submissions/create.html [hunk 4]
```diff
@@ -80,6 +85,7 @@ <h1 class="busta shout">{% block view_title %}Submit Your Game{% endblock %}</h1
                     <label for="id_team_desciption" class="announce">
```
Comment (andrewhayward) [inline gameon/submissions/templates/submissions/create.html:85]: Another `description` - if you felt like changing them.

File: gameon/submissions/templates/submissions/create.html [hunk 5]
```diff
@@ -80,6 +85,7 @@ <h1 class="busta shout">{% block view_title %}Submit Your Game{% endblock %}</h1
...
                         Team bio <span class="opt">(optional)</span>
                         {% for error in form.team_desciption.errors %}<em>{{ error }}</em>{% endfor %}
+                        <span class="extra meta" id="info_team_desciption">(maximum 250 characters)</span>
```
Comment (andrewhayward) [inline gameon/submissions/templates/submissions/create.html:88]: Is it worth point all of these `description` issues out?

File: gameon/submissions/templates/submissions/create.html [hunk 6]
```diff
@@ -106,6 +112,7 @@ <h2 class="whimper">What is Marketplace and why is it awesome?</h2>
 {% endblock %}
 
 {% block page_js %}
+<script src="{{ static('base/js/libs//jquery.textareaCounter.plugin.js') }}"></script>
```
Comment (andrewhayward) [inline gameon/submissions/templates/submissions/create.html]: Random double slash?

File: gameon/submissions/forms.py (hunk 2 above)
Comment (rossbruniges) [inline gameon/submissions/forms.py:18]: I fingers had a list while typing

File: gameon/submissions/templates/submissions/create.html (hunk 5 above)
Comment (rossbruniges) [inline gameon/submissions/templates/submissions/create.html:88]: Yeah, I deserve the shame

**Comment by rossbruniges** [comment]
Have fixed the double slashes - have (for now left desciption as it is). It has been that way for a while, might be a good individual bug.

**Comment by rossbruniges** [comment]
All references to team_desciption are now gone. Let us never mention it again :) https://github.com/rossbruniges/gameon/commit/8504b6152ced35d2810dc189327a11e84bfc920e

**Pull request merged**